ResourceStatusSystem/DB/ResourceManagementDB.py

```diff
@@ -134,65 +143,65 @@ def __init__( self ):
                                Column( 'OpenTickets', Integer, nullable = False, server_default = '0'),
                                Column( 'GocSite', String( 64 ), nullable = False, primary_key = True ),
                                Column( 'Link', String( 1024 ), nullable = False ),
-                               Column( 'LastCheckTime', DateTime, nullable = False ),
+                               Column( 'LastCheckTime', DateTime, nullable = True ),
                                mysql_engine = 'InnoDB' )
 
     JobCache                 = Table( 'JobCache', self.metadata,
                                Column( 'Status', String( 16 ), nullable = False ),
-                               Column( 'Efficiency', DOUBLE, nullable = False, server_default = '0'),
+                               Column( 'Efficiency', FLOAT, nullable = False, server_default = '0'),
```

Comment:
I'm not sure if it becomes "DOUBLE" when it creates the schema (I'll check it to be sure)
However it fixed my problem, for some reason when I had set "DOUBLE" the results from some rows where `DECIMAL(1,1)` instead of `1.1`.

Maybe it was happening because I imported the wrong "DOUBLE" type from sqlalchemy, I will double check it to be sure.